Client/src/main/java/org/projectcardboard/client/controller/ResourcesCenter.java

```diff
@@ -15,7 +15,8 @@
 
 public class ResourcesCenter {
   private final static ResourcesCenter ourInstance = new ResourcesCenter();
-  private static final String PATH = "Client/src/main/resources";
+  private static final String PATH = "Client/src/main/resources"; // todo read in as
```

Comment:
Wouldn't do it here, but yeah all the `File(PATH)` stuff could be switched to using the resources API since it's already in the right place.